### ISSUE TITLE
ASCII Protocol

### ISSUE THREAD

**Issue by thecodeboss** [ananthakumaran/memcachex#13]
We use mcrouter for our memcache setup, but unfortunately it [only supports ascii protocol](https://github.com/facebook/mcrouter/issues/6).

I'm thinking of writing an ascii protocol implementation for Memcachex; is something that y'all would be onboard with?

**Comment by ananthakumaran** [comment]
Yes, provided it doesn't cause any performance issue with binary protocol and the implementation is cleanly separated. I hadn't given much thought about implementing multiple protocols, so not sure how easy or hard it would be to implement it with the current structure.
